apm-agent-tracer/src/main/java/co/elastic/apm/agent/tracer/metadata/BodyCapture.java

```diff
@@ -0,0 +1,47 @@
+package co.elastic.apm.agent.tracer.metadata;
+
+import javax.annotation.Nullable;
+
+public interface BodyCapture {
```

Comment:
Why is there no access to the captured body from the interface?

Reply:
This is intentional: Instrumentations only need to write it, it is not intended to be read back by them.